Example_Programs/calculator_complex/README.md

```diff
@@ -0,0 +1,29 @@
+# Time to code
+## Problem Statement
+* Write functions to compute **Sum**, **Difference**, **Product** and **Division** of two **Complex numbers** of the below structure.
+```
+typedef struct complex_t {
+    float real;
+    float imaginary;
+} complex_t;
+```
+
+## Guidelines:
+### Programming
+* **Do not use scanf functions**. Assume test values for arguments to functions.
+* Implement different functions for each operation instead of writing everything under main function.
+* Create a **header file**,  **source code file** and **test file**.
+* Use **Dynamic memory** wherever possible.
+
+### Tool usage
+* Write **Unit test** cases for each function.
+* **Makefile** for **compilation**, **running test** cases and additional targets.
+* Create **Documentation** using **Doxygen**.
+* Use **cppcheck** and **Valgrind** tools to analyze the code.
+
+## Resources:
+* [Structure in C: you should know in depth](https://aticleworld.com/structure-in-c/)
+
+## Solutions
+
+* Repo link here
```

Comment:
Add your repo lin and name in the below syntax:
* Solution by [Your name](link to your repo)